nomad-clouddriver/src/main/kotlin/io/armory/plugin/nomad/descriptions/UpsertNomadJobDescription.kt

```diff
@@ -3,6 +3,9 @@ package io.armory.plugin.nomad.descriptions
 import com.hashicorp.nomad.apimodel.Job
 
 data class UpsertNomadJobDescription(
-        var job: Job? = null,
-        var credentials: String? = null) {
+  /**
```

Comment:
This is just to get the Jackson marshalling to work for a data class. Not what I would normally want to do.

Reply:
Oh, I see. If we add the `KotlinModule` to Jackson, it adds better support for data classes so this isn't necessary.

Reply:
Yes, wanted to just get an existing ObjectMapper injected, but there isn't one available unless aws is enabled. I also tried using the kotlin module in the ObjectMapper I create in the plugin, but that caused some weird issues where the plugin wouldn't run. I should probably look into that further.